monome/teletype
# Re-add commits lost due to squashing of PR #114

#### What does this PR do?

Adds the commits lost due to the accidental squashing of PR #114

#### How should this be manually tested?

Doesn't need hardware testing as it makes no changes. But anyone interested can look over the commits and try to check this _actually_ re-adds the commits in the way that we want.

----

Can I get a sign of from:

 - @tehn 
 - @scanner-darkly 
 - @burnsauce 

before I merge this please. (Also, attn: @cmcavoy  and @jlmitch5.)

To the best of my knowledge, this will add back in the missing commits by creating 2 parallel branches of code that end up in more or less the same place. `git blame` appears to show the "individual commit branch" as it's more recent. My expectation is that `git bisect` will be usable for tracking down issues that haven arisen as part of the 2.1 process, but that care will have to be taken when choosing the start and end commits.
